UnityProject/Assets/Scripts/Mobs/AI/MobAI.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+
+namespace Mobs.AI
+{
+	public class MobAI : MonoBehaviour
```

Comment:
there is now ready system for MobAI? 
with the contemplate goal and stuff

Reply:
Someone removed it sometime ago, and I couldn't find it again. So I took the simplest approach to finte state machine, we don't need the fancy visuals; just a gameObject to list all states and a way to keep track of all the active ones.

Reply:
Public class MobController : MonoBehaviour
it still there

Reply:
it will be deleted when we move all mobs to V2

Reply:
The question is which is better?

Reply:
mine

Reply:
Need discussion with everyone,  To make sure it's okay